kafka/0.5/Dockerfile

```diff
@@ -35,11 +43,9 @@ RUN echo "$MD5HASH  /tmp/kafka.tgz" | md5sum -c - &&\
 COPY ./log4j.properties $KAFKA_HOME/config/log4j.properties
 
 #
-# Change ownership and switch user
+# Allow random UID to use Kafka
 #
-RUN chown -R kafka $KAFKA_HOME && \
-    chgrp -R kafka $KAFKA_HOME
-USER kafka
+RUN chmod -R g+w,o+w $KAFKA_HOME
```

Comment:
Not sure if I follow you - my build log is
```
Step 11 : RUN echo "$MD5HASH  /tmp/kafka.tgz" | md5sum -c - &&    tar -xzf /tmp/kafka.tgz -C $KAFKA_HOME --strip-components 1 &&    rm -f /tmp/kafka.tgz
 ---> Running in 7cf6a342c41b
/tmp/kafka.tgz: OK
 ---> 7dfbcc9a1e1d
Removing intermediate container 7cf6a342c41b
Step 12 : COPY ./log4j.properties $KAFKA_HOME/config/log4j.properties
 ---> 60bb8e30fcf2
Removing intermediate container 0fdeee4e5828
Step 13 : RUN chmod -R g+w,o+w $KAFKA_HOME
 ---> Running in 83fb5880dee9
 ---> 610ccc55fbec
Removing intermediate container 83fb5880dee9
Step 14 : WORKDIR $KAFKA_HOME
 ---> Running in d0e6230024a1
 ---> d6426465fcf6
Removing intermediate container d0e6230024a1
Step 15 : EXPOSE 9092
 ---> Running in f5b1d7cbb61a
 ---> 75c69c95bdaf
```
I do not see any issue with changing the privileges. Just in case, my Docker env is
```
docker version
Client:
 Version:      1.12.6
 API version:  1.24
 Go version:   go1.6.2
 Git commit:   78d1802
 Built:        Tue Jan 31 23:35:14 2017
 OS/Arch:      linux/amd64

Server:
 Version:      1.12.6
 API version:  1.24
 Go version:   go1.6.2
 Git commit:   78d1802
 Built:        Tue Jan 31 23:35:14 2017
 OS/Arch:      linux/amd64
```